### ISSUE TITLE
New terms need to be added

### ISSUE THREAD

**Issue by linikujp** [ClinicalTrialOntology/CTO#17]
1. clinical trial enrollment 
2. clinical trial registry
3. clinical trial participant
4. human subject (OPMI)
5. enrolled patient
6. placebo medical intervention
7. healthy enrollee
8. outcome measurement datum
9. investigation collaborator role (OPMI will donate)
10. clinical trial sponsor  role (OPMI will donate)
11. central contact person (OPMI will donate)
12. facility contact (OPMI will donate)
13. contact for public queries (OPMI will donate)
14. contact for scientific queries (OPMI will donate)
15. contact person role (OPMI)
16. contact person information (OPMI)
17. drug clinical trial
18. investigational molecular entity
-- from ClinicalTrials,gov--
19. vaccine clinical trial
20. behavioral clinical trial
21. biological clinical trial
22. combination product clinical trial
23. device clinical trial OR medical device clinical trial?
24. diagnostic test clinical trial
25. dietary supplement clinical trial
26. genetic clinical trial
27. procedure clinical trial
--from FDA: types of clinical research--
28. Treatment Research : generally involves an intervention such as medication, psychotherapy, new devices, or new approaches to surgery or radiation therapy. 
29. Prevention Research: looks for better ways to prevent disorders from developing or returning. Different kinds of prevention research may study medicines, vitamins, vaccines, minerals, or lifestyle changes. 
30. Diagnostic Research: refers to the practice of looking for better ways to identify a particular disorder or condition. 
31. Screening Research: aims to find the best ways to detect certain disorders or health conditions. 
32. Quality of Life Research: explores ways to improve comfort and the quality of life for individuals with a chronic illness. 
33. Genetic studies: aim to improve the prediction of disorders by identifying and understanding how genes and illnesses may be related. Research in this area may explore ways in which a person’s genes make him or her more or less likely to develop a disorder. This may lead to development of tailor-made treatments based on a patient’s genetic make-up. 
34. Epidemiological studies: seek to identify the patterns, causes, and control of disorders in groups of people. 

**Objective properties**
1. has active ingredient
2. investigates patients administered
3. investigated agent in

**Comment by linikujp** [comment]
No. 15. contact person role should stay in OPMI, as it is not clinical trial specific.

**Comment by yongqunh** [comment]
OK. Will work on this later with Assiyah. Oliver

**Comment by sgebel** [comment]
1. clinical trial enrollment
added to CTO (updates) w/o definition  
2. clinical trial registry
exists already in CTO     
3. clinical trial participant
added to CTO (updates) w/o definition
4. human subject (OPMI)
added to CTO (updates) imported from OPMI
5. enrolled patient
added to CTO (updates) w/o definition
6. placebo medical intervention
added to CTO (updates) w/o definition
7. healthy enrollee
added to CTO (updates) w/o definition
8. outcome measurement datum
added to CTO (updates) w/o definition
8.1. primary outcome measurement datum
added to CTO (updates) w/o definition
8.2. secondary outcome measurement datum
added to CTO (updates) w/o definition

**Comment by linikujp** [comment]
Adding new relations:
1. Please import RO  http://purl.obolibrary.org/obo/RO_0002248 for 'has active ingredient'.
